test/Testing/LeanCode.IntegrationTestHelpers.Tests/ConfigurationOverridesTests.cs

```diff
@@ -0,0 +1,183 @@
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace LeanCode.IntegrationTestHelpers.Tests;
+
+public class ConfigurationOverridesTests
+{
+    [Fact]
+    public void Empty_ConfigurationOverrides_Results_In_Empty_Configuration_test()
+    {
+        var configuration = new ConfigurationBuilder().Add(new ConfigurationOverrides()).Build();
+
+        Assert.Empty(configuration.AsEnumerable());
+    }
+
+    [Fact]
+    public void ConfigurationOverrides_single_Add_method_test()
+    {
+        var overrides = new ConfigurationOverrides().Add("key", "value");
+
+        var configuration = new ConfigurationBuilder().Add(overrides).Build();
```

Comment:
Let's wrap the `new ConfigurationBuilder().Add(overrides).Build()` in a method called `BuildConfiguration(ConfigurationOverrides overrides)` - will make things cleaner.